### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by adel-gu**
### Project Requirments Accomplished 🎯

- Build portfolio website Responsive for different devices screens.
- Use media queries.
- Grid and Flexbox were used to achieve the layouts.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,160 +55,246 @@ <h1>Adel Guitoun</h1>
         </div>
```
Comment (ElsonOtake) [inline index.html:55]: - [x] I think it's better to use the text `Start collaboration` on the button. Stick to the Figma template is a project requirement.

File: index.html [hunk 2]
```diff
@@ -310,7 +412,9 @@ <h2 class="primary-heading">Get started</h2>
               </ul>
```
Comment (ElsonOtake) [inline index.html:413]: - [x] I think it's better to use the text `Start collaboration` on the button. Stick to the Figma template is a project requirement.

File: index.html [hunk 3]
```diff
@@ -41,160 +55,246 @@ <h1>Adel Guitoun</h1>
...
+            />
+          </li>
+
+          <li class="project-item pos-relative flex direction span-2">
+            <div class="project-card pos-relative bottom-pos">
+              <div class="project-img">
                 <img
-                  src="./assets/icons/arrow-button.svg"
-                  alt="see this project"
+                  src="./assets/images/projects-section/project4.png"
```
Comment (ElsonOtake) [inline index.html:272]: - [x] I think it's better to reduce the size of this image to stick to the Figma template. Use a width of 30%. Stick to the template is a project requirement.

File: index.html [hunk 4]
```diff
@@ -41,160 +55,246 @@ <h1>Adel Guitoun</h1>
...
-              </button>
+              </div>
+              <div class="container project-details pos-absolute bottom-pos">
+                <h3>Project name goes here</h3>
+                <ul role="list" class="features-container flex">
+                  <li class="feature">HTML/CSS</li>
+                  <li class="feature">Ruby on Rails</li>
+                  <li class="feature">JavaScript</li>
+                </ul>
+                <button class="btn secondary-btn flex">
```
Comment (ElsonOtake) [inline index.html:94]: - [x] Please fix the style for the button when responsive. The button is touching the border. Your project must look amazing for any size of the screen.
Screen Shot 2022-08-27 at 11 46 00 AM

File: index.html [hunk 5]
```diff
@@ -41,160 +55,246 @@ <h1>Adel Guitoun</h1>
...
         <ul role="list" class="projects-lists grid">
-          <li class="project-item flex direction">
-            <div class="project-img">
-              <img
-                src="./assets/images/projects-section/project1.png"
-                alt="Project 1"
-              />
+          <li class="project-heading-container">
+            <div class="container">
+              <h2 class="primary-heading">Projects</h2>
```
Comment (ElsonOtake) [inline index.html:76]: - [x] Please check the style of text "Project" when responsive. Your portfolio must look amazing for any size of screen.
Screen Shot 2022-08-27 at 11 49 55 AM

File: index.html [hunk 6]
```diff
@@ -203,7 +303,7 @@ <h3>Project name goes here</h3>
       <section class="section about-section">
         <div class="container about-container flex direction">
           <div class="about-content">
-            <h2 class="primary-heading">About</h2>
+            <h2>About</h2>
```
Comment (ElsonOtake) [inline index.html:307]: - [x] Please correct the styling for the "Get my CV" button when responsive. Your portfolio must look amazing for any size of screen.
Screen Shot 2022-08-27 at 11 53 28 AM

File: index.html [hunk 7]
```diff
@@ -246,17 +346,19 @@ <h4>Skills</h4>
...
 
       <!-- Footer Section -->
       <footer class="section footer-section bg-decoration">
-        <div class="container">
+        <div class="container grid">
+          <div class="top-bg-decoration hide"></div>
           <div class="top-footer-container flex direction">
             <div class="contact-content">
-              <h2 class="primary-heading">Get started</h2>
+              <h2>Get started</h2>
```
Comment (ElsonOtake) [inline index.html:362]: - [x] Please correct the style for the "Get started". It's creating a margin when responsive. Your portfolio must look amazing for any size of screen.
Screen Shot 2022-08-27 at 11 59 54 AM

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @adel-gu,

Good job so far! 🎉 🎉 
Nice work!! 🏆 🏆 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] I think it's better to use imperative tense and capitalized your commit messages. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).
- [OPTIONAL] Kindly add a full descriptive README file to your project - please use all fields of [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. Always think about people you share your code: collaborators, code reviewers, programmers, recruiters, or non-technical users who might want to reuse or improve your project.

Your project will look very professional and certainly you will be very proud of it.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (adel-gu) [reply index.html:272]: Hi @ElsonOtake, please can you explain more about this issue,

Thank you.

**Review by adel-gu — COMMENTED** [review]
Hi @ElsonOtake 👋

**Review by franclobo — APPROVED** [review]
Hi @adel-gu, 

# STATUS: APPROVED 👏🏽 👏🏽 👏🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Professional README file ✔️ 
- Good communication with the reviewer 💯 
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**